mender-convert-functions.sh

```diff
@@ -1,12 +1,14 @@
 #!/bin/bash
 
-# Partition alignment value in bytes (8MB).
-partition_alignment=8388608
+# Partition alignment value in bytes (4MB).
+partition_alignment=4194304
```

Comment:
In that way you have affected both BBB & RPI3. Alignment equal to 4MB should apply only to RPI3.